### PR TITLE
Changes various event overrides

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

image

As requested.
## How This Contributes To The Bubbers Roleplay Experience

Host request.

## Changelog

:cl: Cursor
balance: Lone-Ops and Bureaucratic error are back.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 1]
```diff
@@ -57,8 +57,8 @@
  * Upped to ensure lowpop steamroll does not happen
  */
 /datum/round_event_control/spider_infestation
-	// min_players = 70
-	max_occurrences = 0
+	min_players = 40 //ZUBBERS EDIT CHANGE, ORIGINAL: //min_players = 70
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:60]:
```suggestion
	min_players = 40 //BUBBERS EDIT CHANGE, ORIGINAL: //min_players = 70
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 2]
```diff
@@ -57,8 +57,8 @@
...
+	max_occurrences = 1 //ZUBBERS EDIT CHANGE, ORIGINAL: //max_occurrences = 0
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:61]:
```suggestion
	max_occurrences = 1 //BUBBERS EDIT CHANGE, ORIGINAL: //max_occurrences = 0
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 3]
```diff
@@ -95,8 +95,11 @@
  * Removed:
  * Does not have policy. Will re-add if/when policy is added
  */
-/datum/round_event_control/operative
-	max_occurrences = 0
+// ZUBBERS EDIT BEGIN
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:98]:
```suggestion
// BUBBERS EDIT BEGIN
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 4]
```diff
@@ -95,8 +95,11 @@
...
+/* /datum/round_event_control/operative
+/	max_occurrences = 0
+*/
+//* ZUBBERS EDIT END.
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:102]:
```suggestion
//* BUBBERS EDIT END.
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 5]
```diff
@@ -132,5 +135,9 @@
  * Removed:
  * Paperwork Error is too intrusive and should be staff-only.
  */
+/* ZUBBERS EDIT START
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:138]:
```suggestion
/* BUBBERS EDIT START
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 6]
```diff
@@ -132,5 +135,9 @@
...
 /datum/round_event_control/bureaucratic_error
 	max_occurrences = 0
+*/
+// ZUBBERS EDIT END
```
Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:142]:
```suggestion
// BUBBERS EDIT END
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 7]
```diff
@@ -57,8 +57,8 @@
  * Upped to ensure lowpop steamroll does not happen
  */
 /datum/round_event_control/spider_infestation
-	// min_players = 70
-	max_occurrences = 0
+	min_players = 40 //BUBBERS EDIT CHANGE, ORIGINAL: //min_players = 70
```
Comment (Cyprex) [inline modular_skyrat/modules/events/code/event_overrides.dm:60]: Leave these as is, we had this once(via the dynamic ruleset) and it was pretty disastrous.

Comment (theselfish) [inline modular_skyrat/modules/events/code/event_overrides.dm:60]:
```suggestion
	min_players = 70
```

**Comment by Cyprex** [comment]
Good to merge once CI passes

**Comment by Cyprex** [comment]
If you could change the CL to "Two events are back: Lone ops when there's 20 or more players, and bureaucratic error(at any pop)

**Comment by theselfish** [comment]
@Cyprex Fixed;.

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 8]
```diff
@@ -96,7 +96,8 @@
  * Does not have policy. Will re-add if/when policy is added
  */
 /datum/round_event_control/operative
-	max_occurrences = 0
+	//max_occurrences = 0
```
Comment (KathrinBailey) [inline modular_skyrat/modules/events/code/event_overrides.dm:99]: why comment out this line?

Comment (theselfish) [reply modular_skyrat/modules/events/code/event_overrides.dm:99]: Commenting out is better, in case the file is updated. Less compatibility headaches.

**Pull request merged**